scripts/install

```diff
@@ -40,6 +40,9 @@ ynh_add_config --template="domain.cfg.lua" --destination="$install_dir/conf/conf
 #=================================================
 ynh_script_progression --message="Configuring Nginx for muc and xmpp-upload domains..."
 
+# nginx needs read access to metronome files for HTTP upload
+usermod -a -G "$app" www-data
```

Comment:
Does it really solves any issue ? To me the issue is that the /home/yunohost.app/metronome should have "www-data" as group ?

Adding an user to a group doesn't imply that files will be created/readable by this group ... unless maybe you define it as primary group ... but anyway this is not great from a security point of view : www-data should be able to read the files created by metronome, but metronome shouldn't be able to read every file that www-data can access ...

Reply:
> www-data should be able to read the files created by metronome, but metronome shouldn't be able to read every file that www-data can access ...

That's exactly what adding *user* `www-data` to `metronome` *group* does. Not the other way around :)

> To me the issue is that the /home/yunohost.app/metronome should have "www-data" as group ?

bullseye was using the sgid bit [here](https://github.com/YunoHost/yunohost/blob/eb14e404d6fdcb940f8e6048dc0a7c2799c4e75c/hooks/conf_regen/12-metronome#L65C9-L65C54) so i ended up doing the same. Is it better?

Reply:
>That's exactly what adding user www-data to metronome group does. Not the other way around :)

Ooooh my bad ... But that's still an issue because then www-data may have access to secrets in Metronome's conf ... or maybe there's no secret in the case of Metronome ...

>bullseye was using the sgid bit [here](https://github.com/YunoHost/yunohost/blob/eb14e404d6fdcb940f8e6048dc0a7c2799c4e75c/hooks/conf_regen/12-metronome#L65C9-L65C54) so i ended up doing the same. Is it better?

Yeah that sounds definitely like the right way to proceed